packages/backend-core/src/logging/pino/logger.ts

```diff
@@ -31,6 +42,15 @@ if (env.isDev()) {
 
 export const logger = pino(pinoOptions)
 
+export function disableLogger() {
+  console.log = BUILT_INS.log
```

Comment:
How come this is disabling the logs? Is the BUILT_INS.log not the console.log?

Reply:
By default, all of the loggers have been overwritten with the pino logger, if you load backend-core this is the way it works. I didn't want to change this API, but I need the `console.log` to be the standard built in Node.js one *purely* for the CLI, all other usages of backend-core should have their logger replaced by the pino logger.